app/shared/actions/wallet.js

```diff
@@ -70,11 +69,11 @@ export function unlockWallet(wallet, password) {
 
 function encrypt(msg, pass) {
   const keySize = 256;
-  const iterations = 10000;
+  const iterations = 4500;
```

Comment:
Was the change in number of iterations recommended as well?

Reply:
Yep, sure was. The increase in the keySize caused the time to encrypt/decrypt to jump a pretty large amount, so decreasing the iterations helped reduce the overall time to compensate.

Reply:
ok I guess I missed that lol 😄 Thanks for explaining!